Code/CryMP/Client/Client.h

```diff
@@ -67,6 +67,7 @@ class Client : public IGameFrameworkListener, public ILevelSystemListener, publi
 	{
 		std::string key;
 		std::string command;
+		HSCRIPTFUNCTION function = nullptr;
```

Comment:
```suggestion
		SmartScriptFunction function;
```
And then remove all `ReleaseFunc` calls.